scripts/compile_coq.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/sh -e
+
+help() {
+  echo "./compile-coq <compiler path> <path to directory> [--opam opam] [-c|--compile-opam] [-r|--remove]"
+  echo "[compiler path]     is the path to the ocaml compiler"
```

Comment:
I'd rather say it's the path to the compiler repository to test, rather that the compiler (compiler is a bit ambiguous, it could refer to e.g. the `ocamlopt` binary), and maybe specify it uses the files as in, and not the head of the git branch (mainly because that last behavior is the default when opam pins a local packages, and it's better to specify things explicitly).

Reply:
Clarified the explanation.